djmoney/__init__.py

```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 
-__version__ = '0.15.dev'
-default_app_config = 'djmoney.apps.MoneyConfig'
+__version__ = "0.15.dev"
+default_app_config = "djmoney.apps.MoneyConfig"
```

Comment:
Can black be made to not care about quotes? It's going to be annoying :)

Reply:
See https://github.com/python/black#strings

The whole point of Black is about not being configurable.

Reply:
Nope, there is only one option :) It is almost uncompromising :) There were some good arguments for always using double quotes in one issue in black, but I don't remember where exactly. Usually, I set up a pre-commit hook for black and everything works without hassle

Reply:
aha, it is exactly what I was thinking about :)

Reply:
I (almost) always used double quotes for variable/human strings and single quotes for keys and other sorts of constant-ish strings :)